src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1436,6 +1453,17 @@ PUBLIC_API STATUS freePeerConnection(PRtcPeerConnection*);
  */
 PUBLIC_API STATUS peerConnectionOnIceCandidate(PRtcPeerConnection, UINT64, RtcOnIceCandidate);
 
+/**
+ * @brief Set a callback for transport-wide sender bandwidth estimation results
+ *
+ * @param[in] PRtcPeerConnection Initialized RtcPeerConnection
+ * @param[in] UINT64 User customData that will be passed along when RtcOnSenderBandwidthEstimation is called
+ * @param[in] RtcOnSenderBandwidthEstimation User RtcOnSenderBandwidthEstimation callback
+ *
+ * @return STATUS code of the execution. STATUS_SUCCESS on success
+ */
+PUBLIC_API STATUS peerConnectionOnSenderBandwidthEstimation(PRtcPeerConnection, UINT64, RtcOnSenderBandwidthEstimation);
```

Comment:
FYI. Great job on the new API and sample implementation. I am sure this is going to be very useful. :)